varname.py

```diff
@@ -28,8 +28,8 @@ def _get_node(caller):
     When the node can not be retrieved, try to return the first statement.
     """
     try:
-        frame = inspect.stack()[caller+2].frame
-    except IndexError:
+        frame = sys._getframe(caller + 2)
+    except Exception:
```

Comment:
Any specific types of exceptions can be predicted here?

Reply:
If the argument is too big you get `ValueError: call stack is not deep enough` in CPython. But maybe other implementations raise something different, it's probably not well specified. And I think in some implementations it doesn't exist so you get AttributeError.

Reply:
Fair enough. Let's keep it as `Exception` since `frame` falls back to `None` if anything fails.